src/UI/CardSelector.hs

```diff
@@ -66,12 +72,12 @@ drawMenu s =
   withBorderStyle unicodeRounded $
   border $
   hLimitPercent 60 $
-  title <=>
+  hCenter (title <+> shuffledWidget s) <=>
```

Comment:
Doing `hCenter` instead of the `hCenteredStrWrap` is good, but note that the text now does not wrap anymore. Doing the same layout with `hCenteredStrWrap` would be possible when concatenating the "(Shuffled)" and the title strings but then you wouldn't be able to make them different colors. So it's fine how it is now, there are probably very few people that have their terminal window that small anyways.